README.md

```diff
@@ -2,6 +2,20 @@
 
 Content Tagger is the publishing application for the [Topic Taxonomy](https://docs.publishing.service.gov.uk/manual/taxonomy.html) on GOV.UK. It also provides some functionality for tagging content in bulk.
 
+### User account types and access permissions
```

Comment:
I'd move this into a `## Further documentation` section below the `## Technical documentation` section.

Reply:
👍 yes, makes sense